var/spack/repos/builtin/packages/rename/package.py

```diff
@@ -0,0 +1,45 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Rename(Package):
+    """Perl-powered file rename script with many helpful built-ins."""
+
+    homepage = "http://plasmasturm.org/code/rename"
+    url      = "https://github.com/ap/rename/archive/v1.600.tar.gz"
+
+    version('1.600', '91beb555c93d407420b5dad191069bb3')
+
+    depends_on('perl', type='nolink')
```

Comment:
`pod2man` is part of the Perl core.  If you have `perl`, you should have `pod2man`.  Unless, perhaps you installed your `perl` from some vendor's packages that disassembled things and package various bits separately.

The Spack perl packages install it:
```
[hartzelg@spack-0 rss]$ find spack-test-drive-2525/ | grep pod2man
spack-test-drive-2525/opt/spack/linux-centos7-x86_64/gcc-5.4.0/perl-5.24.0-xjz2cnt45rbcfpawhknn6lhd5wuuz3ra/bin/pod2man
spack-test-drive-2525/opt/spack/linux-centos7-x86_64/gcc-5.4.0/perl-5.24.0-xjz2cnt45rbcfpawhknn6lhd5wuuz3ra/man/man1/pod2man.1
spack-test-drive-2525/opt/spack/linux-centos7-x86_64/gcc-5.4.0/perl-5.22.2-secuh2wuhq3a3huxgnuplmfqc225pcht/bin/pod2man
spack-test-drive-2525/opt/spack/linux-centos7-x86_64/gcc-5.4.0/perl-5.22.2-secuh2wuhq3a3huxgnuplmfqc225pcht/man/man1/pod2man.1
```